// Copyright 2022 iLogtail Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";
package sls_logs;

message LogtailBufferMeta 
{
    required string project = 1;
    required string endpoint = 2;
    required string aliuid = 3;
    optional string logstore = 4;
    optional int32 datatype = 5;
    optional int32 rawsize = 6;
    optional string shardhashkey = 7;
}
